versions.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 4.57.0"
+      version = ">= 5.11.0"
```

Comment:
Is there a reason to up the minimum version to v5.11.0? Are you using functionality thats not provided in v4.57.0?